sensors/ev3_uart_sensor_defs.c

```diff
@@ -109,7 +109,7 @@ const struct ev3_uart_sensor_info ev3_uart_sensor_defs[] = {
 				 * the EV3 brick. As a result, the sensor will time out and reset.
 				 *
 				 * @name_footnote: [^cal-mode]
-				 * @description: Calibration ???<br />All LEDs rapidly cycling, appears white
+				 * @description: Calibration ???<br />Red LED flashing every 4 seconds, then goes continous
```

Comment:
I don't know if it is important, but it resets after a random period of time each time I try. It's around 4-20 blinks, but I'm not sure.